config/vehicle-config.js

```diff
@@ -0,0 +1,13 @@
+var SettingsLoader=require('../app/util/PersistentSettings');
+
+var vehicle_config = {
+  fixed_wings: {
+    throttle: "ch1_in",
+    roll: "ch2_in",
+    pitch: "ch3_in",
+    rudder_yaw: "ch4_in",
+    autopilot_ON_OFF: "ch5_in",
```

Comment:
keep the naming convention consistent. ie. autopilot_status works just as well in describing that channel